src/sql/strong-oracle/disable.sql

```diff
@@ -0,0 +1,119 @@
+
+DECLARE
+  c_action CONSTANT VARCHAR2(10) := 'DISABLE';
+BEGIN
```

Comment:
I'm not really qualified to review this, but it looks pretty straightforward. In any case, it's just used for testing, right?